lib/plugins/rotate_asg_instances/asg.rb

```diff
@@ -1,20 +1,35 @@
-require 'moonshot/ssh_fork_executor'
-
 module Moonshot
   class ASG # rubocop:disable Metrics/ClassLength
     include Moonshot::CredsHelper
 
-    def initialize(resources)
+    def initialize(resources, ssh_user)
       @resources = resources
       @ilog = @resources.ilog
-      @ssh_user = ENV['MOONSHOT_SSH_USER'] || ENV['LOGNAME']
+      @ssh_user = ssh_user
+      puts "ssh_user: #{@ssh_user}"
```

Comment:
I wonder a puts here would affect the interactive logging format of the tool.

Reply:
That I put for debug purpose actually to check whether CDB_CI_USER injects correctly to plugin. It is setting correctly still rls-test breaks saying failed to ssh. Looking into it.

Reply:
I will remove it 🙂